src/object/srv_ec_aggregate.c

```diff
@@ -1328,9 +1329,9 @@ agg_peer_update_ult(void *arg)
 		D_FREE(ec_agg_in->ea_remove_recxs.ca_arrays);
 		D_FREE(ec_agg_in->ea_remove_eps.ca_arrays);
 	}
+out:
```

Comment:
move to original location

Also, you can remove the if(rpc) below since you assert it above